compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -245,6 +245,28 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         }
         None
     }
+    fn suggest_missing_writer(
```

Comment:
nit, spaces between these methods helps with readability
```suggestion

    fn suggest_missing_writer(
```